templates/partials/todo.blade.php

```diff
@@ -1,18 +1,15 @@
-<div class="grid u-mb-2">
-    <div class="grid-xs-12">
-
-        <table class="table mod-guide-todo-list">
+<div class="o-grid modularity-guide-todos js-modularity-guide-todos">
+    <div class="o-grid-12">
+        <table class="table mod-guide-todo-list js-modularity-guide-todos__table">
             <thead>
                 <tr>
-                    <th class="text-center print-only" width="20">{{_e('Done', 'modularity-guides')}}</th>
                     <th>{{_e('Title', 'modularity-guides')}}</th>
                     <th>{{_e('Link', 'modularity-guides')}}</th>
```

Comment:
Is this translatable without <?php tags? Previously po-edit haven't been able to find these.

Reply:
Old code, I have only modified what did not work. Haven't test this one out though

Reply:
Tested & verified - it works.